One-Electron-Property/IR-Intensities/intensities.py

```diff
@@ -0,0 +1,173 @@
+"""
+A reference implementation for computing IR intensities.
+
+References:
+- Equations from [Yamaguchi:1993]
+- Conversion factors from NIST
+"""
+
+__authors__ = ["Kirk C. Pearce", "Dominic A. Sirianni"]
+__credits__ = ["Kirk C. Pearce" , "Dominic A. Sirianni", 
+               "Andy C. Simmonet", "Zachary  Glick",
+               "C. David Sherrill", "Daniel R. Nascimento"]
+
+__copyright__ = "(c) 2014-2018, The Psi4NumPy Developers"
+__license__ = "BSD-3-Clause"
+__date__ = "2020-11-11"
+
+import numpy as np
+import psi4
+from psi4 import *
+np.set_printoptions(precision=10, linewidth=200, suppress=True)
+psi4.core.set_output_file('output.dat', False)
+
+# Setup
+numpy_memory = 2 
+
+# Useful Constants
+psi_c = psi4.constants.get("Natural unit of velocity") * 100                                                    # speed of light in cm/s
+psi_na = psi4.constants.na                                                                                      # Avogdro's number
+psi_alpha = qcel.constants.get("fine-structure constant")                                                       # finite-structure constant
+
+# Unit Conversions
+#
+# Energy conversions
+hartree2joule = psi4.constants.get("Atomic unit of energy")                                                     # Eh -> J
+#
+# Distance conversions
+psi_bohr2m = psi4.constants.bohr2m                                                                              # bohr -> m
```

Comment:
Could we use these conversions as they come up?

Reply:
As in just define the conversion unit right before using it? If the conversion factor is used in multiple places, but only called before the first use, is it clear enough where it comes from? It very well may be, just want to check before making those changes